rascaline/src/calculators/dummy_calculator.rs

```diff
@@ -75,6 +77,8 @@ impl CalculatorBase for DummyCalculator {
 
     #[allow(clippy::clippy::cast_precision_loss)]
     fn compute(&mut self, systems: &mut [&mut dyn System], descriptor: &mut Descriptor) {
+        // Thes message of this log is tested, therefore if changed here, it has to be changed also in the tests
+        info!{"Computation of DummyCalculator is invoked."}
```

Comment:
```suggestion
        info!("this is an info message used for testing purposes");
        warn!("this is a warning message used for testing purposes");
```

This way we can also check passing the logging level!

Reply:
The current message was chosen to make it more informative , so it can be also considered as info message for the other tests. But even then it was a bit annoying to see it printed out in every test. Therefore I have set the default log level to warnings so one only sees the message when changing the default (I do not checked warnings at the moment).

I think in the long run a dedicated function on the rust side would be the best option. For a temporary solution I can change it the suggestion. I guess if we also want to check the warnings then there is no way of creating an informative message, so what you suggest is currently the best option.

Reply:
changed

Reply:
> I think in the long run a dedicated function on the rust side would be the best option. For a temporary solution I can change it the suggestion. I guess if we also want to check the warnings then there is no way of creating an informative message, so what you suggest is currently the best option.

I had another idea to reduce the noise: you can create a DummyCalculator with the `string` (hyper)-parameters set to `log-test`, and then check the parameter before emitting the log messages.

Reply:
yes seems just like a lot of work for one additional test

Reply:
done